cached_network_image/pubspec.yaml

```diff
@@ -8,16 +8,16 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_cache_manager: ^3.3.0
-  octo_image: ^1.0.0
+  octo_image: ^1.0.2
```

Comment:
There is not really a need to require 1.0.2, 1.0.1 contains a bug fix, so maybe requiring ^1.0.1 makes sense

Reply:
I will set it with ^1.0.1 then.